Would you clean up dependency tree?

<!--

Thanks for stopping by to let us know something could be better!

**PLEASE READ**: If you have a support contract with Google, please create an 
issue in the [support console](https://cloud.google.com/support/) instead of 
filing on GitHub. This will ensure a timely response.

Please run down the following list and make sure you've tried the usual "quick fixes":

  - Search the issues already opened: https://github.com/GoogleCloudPlatform/cloud-sql-jdbc-socket-factory/issues
  - Check for answers on StackOverflow: https://stackoverflow.com/questions/tagged/google-cloud-sql

If you are still having issues, please include as much information as possible:

--> 
## Feature Description

I tried to add the the libraries in this repository into the BOM (https://github.com/GoogleCloudPlatform/cloud-opensource-java/pull/2073) but the tool detected there's transitive dependencies that are not available in Maven Central:

```
xerces:xerces-impl:jar:2.6.2 was not resolved. Dependency path: com.google.cloud.sql:cloud-sql-connector-r2dbc-core:jar:1.2.3 (compile) > io.projectreactor.netty:reactor-netty:jar:1.0.6 (compile) > io.projectreactor.netty:reactor-netty-core:jar:1.0.6 (compile) > io.micrometer:micrometer-core:jar:1.5.0 (compile?) > com.hazelcast:hazelcast:jar:4.0.1 (compile?) > org.codehaus.groovy:groovy-all:jar:2.1.8 (provided?) > com.thoughtworks.xstream:xstream:jar:1.4.2 (compile?) > cglib:cglib-nodep:jar:2.2 (compile?) > ant:ant:jar:1.6.2 (compile?) > xerces:xerces-impl:jar:2.6.2 (compile?)
xml-apis:xml-apis:jar:2.6.2 was not resolved. Dependency path: com.google.cloud.sql:cloud-sql-connector-r2dbc-core:jar:1.2.3 (compile) > io.projectreactor.netty:reactor-netty:jar:1.0.6 (compile) > io.projectreactor.netty:reactor-netty-core:jar:1.0.6 (compile) > io.micrometer:micrometer-core:jar:1.5.0 (compile?) > com.hazelcast:hazelcast:jar:4.0.1 (compile?) > org.codehaus.groovy:groovy-all:jar:2.1.8 (provided?) > com.thoughtworks.xstream:xstream:jar:1.4.2 (compile?) > cglib:cglib-nodep:jar:2.2 (compile?) > ant:ant:jar:1.6.2 (compile?) > xml-apis:xml-apis:jar:2.6.2 (compile?)
maven-plugins:maven-cobertura-plugin:plugin:1.3 was not resolved. Dependency path: com.google.cloud.sql:cloud-sql-connector-r2dbc-core:jar:1.2.3 (compile) > io.projectreactor.netty:reactor-netty:jar:1.0.6 (compile) > io.projectreactor.netty:reactor-netty-core:jar:1.0.6 (compile) > io.micrometer:micrometer-core:jar:1.5.0 (compile?) > ch.qos.logback:logback-classic:jar:1.2.3 (compile?) > org.codehaus.groovy:groovy-all:jar:2.4.0 (compile?) > com.thoughtworks.xstream:xstream:jar:1.4.7 (compile?) > org.jdom:jdom:jar:1.1.3 (compile?) > jaxen:jaxen:jar:1.1.3 (compile?) > dom4j:dom4j:jar:1.6.1 (compile) > jaxen:jaxen:jar:1.1-beta-6 (compile?) > jdom:jdom:jar:1.0 (compile) > jaxen:jaxen:jar:1.0-FCS (compile?) > maven-plugins:maven-cobertura-plugin:plugin:1.3 (compile)
maven-plugins:maven-findbugs-plugin:plugin:1.3.1 was not resolved. Dependency path: com.google.cloud.sql:cloud-sql-connector-r2dbc-core:jar:1.2.3 (compile) > io.projectreactor.netty:reactor-netty:jar:1.0.6 (compile) > io.projectreactor.netty:reactor-netty-core:jar:1.0.6 (compile) > io.micrometer:micrometer-core:jar:1.5.0 (compile?) > ch.qos.logback:logback-classic:jar:1.2.3 (compile?) > org.codehaus.groovy:groovy-all:jar:2.4.0 (compile?) > com.thoughtworks.xstream:xstream:jar:1.4.7 (compile?) > org.jdom:jdom:jar:1.1.3 (compile?) > jaxen:jaxen:jar:1.1.3 (compile?) > dom4j:dom4j:jar:1.6.1 (compile) > jaxen:jaxen:jar:1.1-beta-6 (compile?) > jdom:jdom:jar:1.0 (compile) > jaxen:jaxen:jar:1.0-FCS (compile?) > maven-plugins:maven-findbugs-plugin:plugin:1.3.1 (compile)
com.jcraft:jsch.agentproxy:jar:0.0.6 was not resolved. Dependency path: com.google.cloud.sql:cloud-sql-connector-r2dbc-core:jar:1.2.3 (compile) > io.projectreactor.netty:reactor-netty:jar:1.0.6 (compile) > io.projectreactor.netty:reactor-netty-core:jar:1.0.6 (compile) > io.micrometer:micrometer-core:jar:1.5.0 (compile?) > ch.qos.logback:logback-classic:jar:1.2.3 (compile?) > org.codehaus.groovy:groovy-all:jar:2.4.0 (compile?) > org.apache.ivy:ivy:jar:2.4.0 (compile?) > com.jcraft:jsch.agentproxy:jar:0.0.6 (compile?)
org.jruby.joni:joni:jar:2.1.1-SNAPSHOT was not resolved. Dependency path: com.google.cloud.sql:cloud-sql-connector-r2dbc-core:jar:1.2.3 (compile) > io.projectreactor.netty:reactor-netty:jar:1.0.6 (compile) > io.projectreactor.netty:reactor-netty-core:jar:1.0.6 (compile) > io.micrometer:micrometer-core:jar:1.5.0 (compile?) > com.hazelcast:hazelcast:jar:4.0.1 (compile?) > org.jruby:jruby-complete:jar:1.7.22 (provided?) > org.jruby:jruby-core:jar:noasm:1.7.22 (provided) > org.jruby:yecht:jar:1.0 (compile) > org.jruby:jruby-core:jar:1.7.9 (provided) > org.jruby.joni:joni:jar:2.1.1-SNAPSHOT (compile)

```

Would you cleanup the dependency tree (probably excluding unnecessary dependencies)?

## Alternatives Considered

N/A

## Additional Context

Linkage Checker enforcer rule should reproduce the same error above:

https://github.com/GoogleCloudPlatform/cloud-opensource-java/wiki/Linkage-Checker-Enforcer-Rule

hi @suztomo, I'm trying to reproduce the error you posted by adding the linkage checker to the enforcer rules, but I'm getting a lot of extra errors. This is the result after upgrading to v1.5.9 of `linkage-checker-enforcer-rules`. I thinks the failures might be related to https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/1983
```
[ERROR] Linkage Checker rule found 35 reachable errors:
Class kotlin.jvm.internal.Intrinsics is not found;
  referenced by 4 class files
    reactor.core.publisher.FluxExtensionsKt (io.projectreactor:reactor-core:3.4.6)
    reactor.core.publisher.MonoExtensionsKt (io.projectreactor:reactor-core:3.4.6)
    reactor.core.publisher.MonoWhenFunctionsKt (io.projectreactor:reactor-core:3.4.6)
    reactor.util.function.TupleExtensionsKt (io.projectreactor:reactor-core:3.4.6)
  Cause:
    Unknown
Class kotlin.jvm.functions.Function1 is not found;
  referenced by 3 class files
    reactor.core.publisher.FluxExtensionsKt (io.projectreactor:reactor-core:3.4.6)
    reactor.core.publisher.MonoExtensionsKt (io.projectreactor:reactor-core:3.4.6)
    reactor.core.publisher.MonoWhenFunctionsKt (io.projectreactor:reactor-core:3.4.6)
  Cause:
    Unknown
Class kotlin.jvm.functions.Function0 is not found;
  referenced by 2 class files
    reactor.core.publisher.FluxExtensionsKt (io.projectreactor:reactor-core:3.4.6)
    reactor.core.publisher.MonoExtensionsKt (io.projectreactor:reactor-core:3.4.6)
  Cause:
    Unknown
Class kotlin.jvm.internal.markers.KMappedMarker is not found;
  referenced by 1 class file
    reactor.core.publisher.FluxExtensionsKt (io.projectreactor:reactor-core:3.4.6)
  Cause:
    Unknown
Class kotlin.sequences.Sequence is not found;
  referenced by 1 class file
    reactor.core.publisher.FluxExtensionsKt (io.projectreactor:reactor-core:3.4.6)
  Cause:
    Unknown
Class kotlin.collections.ArraysKt is not found;
  referenced by 2 class files
    reactor.core.publisher.FluxExtensionsKt (io.projectreactor:reactor-core:3.4.6)
    reactor.core.publisher.MonoWhenFunctionsKt (io.projectreactor:reactor-core:3.4.6)
  Cause:
    Unknown
Class kotlin.jvm.JvmClassMappingKt is not found;
  referenced by 2 class files
    reactor.core.publisher.FluxExtensionsKt (io.projectreactor:reactor-core:3.4.6)
    reactor.core.publisher.MonoExtensionsKt (io.projectreactor:reactor-core:3.4.6)
  Cause:
    Unknown
Class io.micrometer.core.instrument.Timer$Sample is not found;
  referenced by 2 class files
    reactor.core.publisher.FluxMetrics (io.projectreactor:reactor-core:3.4.6)
    reactor.core.publisher.MonoMetrics (io.projectreactor:reactor-core:3.4.6)
  Cause:
    Unknown
Class io.micrometer.core.instrument.MeterRegistry is not found;
  referenced by 2 class files
    reactor.core.publisher.FluxMetrics (io.projectreactor:reactor-core:3.4.6)
    reactor.core.scheduler.SchedulerMetricDecorator (io.projectreactor:reactor-core:3.4.6)
  Cause:
    Unknown
Class io.micrometer.core.instrument.Clock is not found;
  referenced by 4 class files
    reactor.core.publisher.FluxMetrics (io.projectreactor:reactor-core:3.4.6)
    reactor.core.publisher.FluxMetricsFuseable (io.projectreactor:reactor-core:3.4.6)
    reactor.core.publisher.MonoMetrics (io.projectreactor:reactor-core:3.4.6)
    reactor.core.publisher.MonoMetricsFuseable (io.projectreactor:reactor-core:3.4.6)
  Cause:
    Unknown
Class io.micrometer.core.instrument.Tags is not found;
  referenced by 2 class files
    reactor.core.publisher.FluxMetrics (io.projectreactor:reactor-core:3.4.6)
    reactor.core.scheduler.SchedulerMetricDecorator (io.projectreactor:reactor-core:3.4.6)
  Cause:
    Unknown
Class io.micrometer.core.instrument.Timer is not found;
  referenced by 5 class files
    reactor.core.publisher.FluxMetrics (io.projectreactor:reactor-core:3.4.6)
    reactor.core.publisher.FluxMetricsFuseable (io.projectreactor:reactor-core:3.4.6)
    reactor.core.publisher.MonoMetrics (io.projectreactor:reactor-core:3.4.6)
    reactor.core.publisher.MonoMetricsFuseable (io.projectreactor:reactor-core:3.4.6)
    reactor.netty.channel.MicrometerChannelMetricsRecorder (io.projectreactor.netty:reactor-netty-core:1.0.7)
  Cause:
    Unknown
    The valid symbol is in io.micrometer:micrometer-core:jar:1.5.0 at io.projectreactor.netty:reactor-netty:1.0.7 (compile) / io.projectreactor.netty:reactor-netty-core:1.0.7 (compile) / io.micrometer:micrometer-core:1.5.0 (compile, optional) but it was not selected because the path contains an optional dependency
Class io.micrometer.core.instrument.Timer$Builder is not found;
  referenced by 2 class files
    reactor.core.publisher.FluxMetrics (io.projectreactor:reactor-core:3.4.6)
    reactor.netty.channel.MicrometerChannelMetricsRecorder (io.projectreactor.netty:reactor-netty-core:1.0.7)
  Cause:
    Unknown
    The valid symbol is in io.micrometer:micrometer-core:jar:1.5.0 at io.projectreactor.netty:reactor-netty:1.0.7 (compile) / io.projectreactor.netty:reactor-netty-core:1.0.7 (compile) / io.micrometer:micrometer-core:1.5.0 (compile, optional) but it was not selected because the path contains an optional dependency
Class io.micrometer.core.instrument.DistributionSummary is not found;
  referenced by 2 class files
    reactor.core.publisher.FluxMetrics (io.projectreactor:reactor-core:3.4.6)
    reactor.netty.channel.MicrometerChannelMetricsRecorder (io.projectreactor.netty:reactor-netty-core:1.0.7)
  Cause:
    Unknown
    The valid symbol is in io.micrometer:micrometer-core:jar:1.5.0 at io.projectreactor.netty:reactor-netty:1.0.7 (compile) / io.projectreactor.netty:reactor-netty-core:1.0.7 (compile) / io.micrometer:micrometer-core:1.5.0 (compile, optional) but it was not selected because the path contains an optional dependency
Class io.micrometer.core.instrument.DistributionSummary$Builder is not found;
  referenced by 2 class files
    reactor.core.publisher.FluxMetrics (io.projectreactor:reactor-core:3.4.6)
    reactor.netty.channel.MicrometerChannelMetricsRecorder (io.projectreactor.netty:reactor-netty-core:1.0.7)
  Cause:
    Unknown
    The valid symbol is in io.micrometer:micrometer-core:jar:1.5.0 at io.projectreactor.netty:reactor-netty:1.0.7 (compile) / io.projectreactor.netty:reactor-netty-core:1.0.7 (compile) / io.micrometer:micrometer-core:1.5.0 (compile, optional) but it was not selected because the path contains an optional dependency
Class io.micrometer.core.instrument.Tag is not found;
  referenced by 1 class file
    reactor.core.publisher.FluxMetrics (io.projectreactor:reactor-core:3.4.6)
  Cause:
    Unknown
Class io.micrometer.core.instrument.Counter is not found;
  referenced by 2 class files
    reactor.core.publisher.FluxMetrics (io.projectreactor:reactor-core:3.4.6)
    reactor.netty.channel.MicrometerChannelMetricsRecorder (io.projectreactor.netty:reactor-netty-core:1.0.7)
  Cause:
    Unknown
    The valid symbol is in io.micrometer:micrometer-core:jar:1.5.0 at io.projectreactor.netty:reactor-netty:1.0.7 (compile) / io.projectreactor.netty:reactor-netty-core:1.0.7 (compile) / io.micrometer:micrometer-core:1.5.0 (compile, optional) but it was not selected because the path contains an optional dependency
Class io.micrometer.core.instrument.Counter$Builder is not found;
  referenced by 2 class files
    reactor.core.publisher.FluxMetrics (io.projectreactor:reactor-core:3.4.6)
    reactor.netty.channel.MicrometerChannelMetricsRecorder (io.projectreactor.netty:reactor-netty-core:1.0.7)
  Cause:
    Unknown
    The valid symbol is in io.micrometer:micrometer-core:jar:1.5.0 at io.projectreactor.netty:reactor-netty:1.0.7 (compile) / io.projectreactor.netty:reactor-netty-core:1.0.7 (compile) / io.micrometer:micrometer-core:1.5.0 (compile, optional) but it was not selected because the path contains an optional dependency
Class kotlin.jvm.internal.FunctionReference is not found;
  referenced by 1 class file
    reactor.core.publisher.MonoExtensionsKt (io.projectreactor:reactor-core:3.4.6)
  Cause:
    Unknown
Class kotlin.jvm.internal.Reflection is not found;
  referenced by 1 class file
    reactor.core.publisher.MonoExtensionsKt (io.projectreactor:reactor-core:3.4.6)
  Cause:
    Unknown
Class kotlin.TypeCastException is not found;
  referenced by 1 class file
    reactor.core.publisher.MonoWhenFunctionsKt (io.projectreactor:reactor-core:3.4.6)
  Cause:
    Unknown
Class reactor.blockhound.integration.BlockHoundIntegration is not found;
  referenced by 1 class file
    reactor.core.scheduler.ReactorBlockHoundIntegration (io.projectreactor:reactor-core:3.4.6)
  Cause:
    Unknown
Class reactor.blockhound.BlockHound$Builder is not found;
  referenced by 1 class file
    reactor.core.scheduler.ReactorBlockHoundIntegration (io.projectreactor:reactor-core:3.4.6)
  Cause:
    Unknown
Class io.micrometer.core.instrument.binder.jvm.ExecutorServiceMetrics is not found;
  referenced by 1 class file
    reactor.core.scheduler.SchedulerMetricDecorator (io.projectreactor:reactor-core:3.4.6)
  Cause:
    Unknown
Class io.micrometer.core.instrument.search.Search is not found;
  referenced by 1 class file
    reactor.core.scheduler.SchedulerMetricDecorator (io.projectreactor:reactor-core:3.4.6)
  Cause:
    Unknown
Class io.micrometer.core.instrument.Metrics is not found;
  referenced by 2 class files
    reactor.util.Metrics (io.projectreactor:reactor-core:3.4.6)
    reactor.netty.Metrics (io.projectreactor.netty:reactor-netty-core:1.0.7)
  Cause:
    Unknown
    The valid symbol is in io.micrometer:micrometer-core:jar:1.5.0 at io.projectreactor.netty:reactor-netty:1.0.7 (compile) / io.projectreactor.netty:reactor-netty-core:1.0.7 (compile) / io.micrometer:micrometer-core:1.5.0 (compile, optional) but it was not selected because the path contains an optional dependency
Class io.micrometer.core.instrument.composite.CompositeMeterRegistry is not found;
  referenced by 1 class file
    reactor.util.Metrics (io.projectreactor:reactor-core:3.4.6)
  Cause:
    Unknown
Class io.micrometer.core.instrument.noop.NoopMeter is not found;
  referenced by 1 class file
    reactor.netty.channel.MicrometerChannelMetricsRecorder (io.projectreactor.netty:reactor-netty-core:1.0.7)
  Cause:
    The valid symbol is in io.micrometer:micrometer-core:jar:1.5.0 at io.projectreactor.netty:reactor-netty:1.0.7 (compile) / io.projectreactor.netty:reactor-netty-core:1.0.7 (compile) / io.micrometer:micrometer-core:1.5.0 (compile, optional) but it was not selected because the path contains an optional dependency
Class io.micrometer.core.instrument.Gauge$Builder is not found;
  referenced by 2 class files
    reactor.netty.resources.MicrometerPooledConnectionProviderMeterRegistrar (io.projectreactor.netty:reactor-netty-core:1.0.7)
    reactor.netty.transport.ByteBufAllocatorMetrics (io.projectreactor.netty:reactor-netty-core:1.0.7)
  Cause:
    The valid symbol is in io.micrometer:micrometer-core:jar:1.5.0 at io.projectreactor.netty:reactor-netty:1.0.7 (compile) / io.projectreactor.netty:reactor-netty-core:1.0.7 (compile) / io.micrometer:micrometer-core:1.5.0 (compile, optional) but it was not selected because the path contains an optional dependency
Class io.micrometer.core.instrument.Gauge is not found;
  referenced by 2 class files
    reactor.netty.resources.MicrometerPooledConnectionProviderMeterRegistrar (io.projectreactor.netty:reactor-netty-core:1.0.7)
    reactor.netty.transport.ByteBufAllocatorMetrics (io.projectreactor.netty:reactor-netty-core:1.0.7)
  Cause:
    The valid symbol is in io.micrometer:micrometer-core:jar:1.5.0 at io.projectreactor.netty:reactor-netty:1.0.7 (compile) / io.projectreactor.netty:reactor-netty-core:1.0.7 (compile) / io.micrometer:micrometer-core:1.5.0 (compile, optional) but it was not selected because the path contains an optional dependency
Class org.conscrypt.HandshakeListener is not found;
  referenced by 1 class file
    io.netty.handler.ssl.ConscryptAlpnSslEngine (io.netty:netty-handler:4.1.63.Final)
  Cause:
    The valid symbol is in org.conscrypt:conscrypt-openjdk-uber:jar:2.5.1 at io.projectreactor.netty:reactor-netty:1.0.7 (compile) / io.projectreactor.netty:reactor-netty-core:1.0.7 (compile) / io.netty:netty-handler:4.1.63.Final (compile) / org.conscrypt:conscrypt-openjdk-uber:2.5.1 (compile, optional) but it was not selected because the path contains an optional dependency
Class org.conscrypt.Conscrypt is not found;
  referenced by 1 class file
    io.netty.handler.ssl.ConscryptAlpnSslEngine (io.netty:netty-handler:4.1.63.Final)
  Cause:
    The valid symbol is in org.conscrypt:conscrypt-openjdk-uber:jar:2.5.1 at io.projectreactor.netty:reactor-netty:1.0.7 (compile) / io.projectreactor.netty:reactor-netty-core:1.0.7 (compile) / io.netty:netty-handler:4.1.63.Final (compile) / org.conscrypt:conscrypt-openjdk-uber:2.5.1 (compile, optional) but it was not selected because the path contains an optional dependency
Class org.conscrypt.AllocatedBuffer is not found;
  referenced by 1 class file
    io.netty.handler.ssl.ConscryptAlpnSslEngine (io.netty:netty-handler:4.1.63.Final)
  Cause:
    The valid symbol is in org.conscrypt:conscrypt-openjdk-uber:jar:2.5.1 at io.projectreactor.netty:reactor-netty:1.0.7 (compile) / io.projectreactor.netty:reactor-netty-core:1.0.7 (compile) / io.netty:netty-handler:4.1.63.Final (compile) / org.conscrypt:conscrypt-openjdk-uber:2.5.1 (compile, optional) but it was not selected because the path contains an optional dependency
Class org.conscrypt.BufferAllocator is not found;
  referenced by 1 class file
    io.netty.handler.ssl.ConscryptAlpnSslEngine (io.netty:netty-handler:4.1.63.Final)
  Cause:
    The valid symbol is in org.conscrypt:conscrypt-openjdk-uber:jar:2.5.1 at io.projectreactor.netty:reactor-netty:1.0.7 (compile) / io.projectreactor.netty:reactor-netty-core:1.0.7 (compile) / io.netty:netty-handler:4.1.63.Final (compile) / org.conscrypt:conscrypt-openjdk-uber:2.5.1 (compile, optional) but it was not selected because the path contains an optional dependency
(io.projectreactor.netty:reactor-netty-core:1.0.7) Interface reactor.netty.resources.LoopResources's method onChannel(io.netty.channel.EventLoopGroup) is not found;
  referenced by 1 class file
    io.r2dbc.postgresql.client.ReactorNettyClient (io.r2dbc:r2dbc-postgresql:0.8.8.RELEASE)
  Cause:
    Dependency conflict: io.projectreactor.netty:reactor-netty:1.0.7 does not define Interface reactor.netty.resources.LoopResources's method onChannel(io.netty.channel.EventLoopGroup) but io.projectreactor.netty:reactor-netty:0.9.20.RELEASE defines it.
      selected: io.projectreactor.netty:reactor-netty:1.0.7 (compile)
      unselected: com.google.cloud.sql:cloud-sql-connector-r2dbc-postgres:1.2.4-SNAPSHOT (compile) / io.r2dbc:r2dbc-postgresql:0.8.8.RELEASE (provided) / io.projectreactor.netty:reactor-netty:0.9.20.RELEASE (compile)
Problematic artifacts in the dependency tree:
io.projectreactor:reactor-core:3.4.6 is at:
  io.projectreactor:reactor-core:3.4.6 (compile)
  and 25 other dependency paths.
io.projectreactor.netty:reactor-netty-core:1.0.7 is at:
  io.projectreactor.netty:reactor-netty:1.0.7 (compile) / io.projectreactor.netty:reactor-netty-core:1.0.7 (compile)
  and 14 other dependency paths.
io.netty:netty-handler:4.1.63.Final is at:
  io.projectreactor.netty:reactor-netty:1.0.7 (compile) / io.projectreactor.netty:reactor-netty-core:1.0.7 (compile) / io.netty:netty-handler:4.1.63.Final (compile)
  and 119 other dependency paths.
io.r2dbc:r2dbc-postgresql:0.8.8.RELEASE is at:
  com.google.cloud.sql:cloud-sql-connector-r2dbc-postgres:1.2.4-SNAPSHOT (compile) / io.r2dbc:r2dbc-postgresql:0.8.8.RELEASE (provided)

```

--- a/r2dbc/core/pom.xml
+++ b/r2dbc/core/pom.xml
@@ -32,6 +32,12 @@
       <groupId>io.projectreactor.netty</groupId>
       <artifactId>reactor-netty</artifactId>
       <version>1.0.7</version>
+      <exclusions>
+        <exclusion>
+          <groupId>io.micrometer</groupId>
+          <artifactId>micrometer-core</artifactId>
+        </exclusion>
+      </exclusions>
     </dependency>
     <dependency>
       <groupId>com.google.cloud.sql</groupId>

